rpm_spec/qubes-qrexec-vm.spec.in

```diff
@@ -48,6 +48,48 @@ Obsoletes:  qubes-core-agent-qrexec < 4.1.0-1
 
 Source0: qubes-core-qrexec-%{version}.tar.gz
 
+%bcond_without selinux
+
+%if %{with selinux}
+Requires: (%{name}-selinux if selinux-policy)
+
+%package selinux
+
+# The following is based on %%selinux_requires, which has the following license:
```

Comment:
Same comment as in https://github.com/QubesOS/qubes-linux-utils/pull/80

Reply:
Same answer (yet again) :laughing: